include/glow/Backends/Backend.h

```diff
@@ -44,10 +44,10 @@ class Backend {
   virtual ~Backend() = default;
 
   /// Prepare the interpreter for execution of new code.
-  virtual void init() = 0;
+  virtual void init(std::unique_ptr<const IRFunction> IR) = 0;
 
   /// Save the bundle for a later standalone execution.
-  virtual void save(llvm::StringRef outputDir) {
+  virtual void save(std::unique_ptr<const IRFunction> IR, llvm::StringRef outputDir) {
```

Comment:
Presumably, this would become a method on `CompiledFunction`, but this is a fine intermediate step.